### PR TITLE
Fix cypress workflow

### PR DESCRIPTION
**Pull request by lezzago**
*Issue #, if available:*
N/A
*Description of changes:*
* Fix cypress workflow
* Add release notes

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
have the right to submit it under the open source license
indicated in the file; or

(b) The contribution is based upon previous work that, to the best
of my knowledge, is covered under an appropriate open source
license and I have the right under that license to submit that
work with modifications, whether created in whole or in part
by me, under the same open source license (unless I am
permitted to submit under a different license), as indicated
in the file; or

(c) The contribution was provided directly to me by some other
person who certified (a), (b) or (c) and I have not modified
it.

(d) I understand and agree that this project and the contribution
are public and that a record of the contribution (including all
personal information I submit with it, including my sign-off) is
maintained indefinitely and may be redistributed consistent with
this project or the open source license(s) involved.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* Fix cypress workflow ([#8](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/8))
+
+### Documentation
+
+* Adds new license header ([#155](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/6))
+
+### Refactoring
+
+* Migrates plugin to OpenSearch Dashboards ([#1](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/1))
+* Cleans up a few more references ([#150](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/5))
```
Comment (thalurur) [inline release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md:17]: number is incorrect? #150 -> #5

File: release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Compatible with OpenSearch-Dashboards 1.0.0
+
+### Infrastructure
+
+* Updates versions and fixes the unit test workflow ([#7](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/7))
+* Fix cypress workflow ([#8](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/8))
+
+### Documentation
+
+* Adds new license header ([#155](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/6))
```
Comment (thalurur) [inline release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md:12]: number seems incorrect? #155 -> #6

**Review by thalurur — APPROVED** [review]

File: release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md (hunk 2 above)
Comment (lezzago) [reply release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md]: thanks, fixed

File: release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md (hunk 1 above)
Comment (lezzago) [reply release-notes/opendistro-for-elasticsearch-index-management-dashboards-plugin.release-notes-1.0.0.0-beta1.md]: thanks, fixed

**Pull request merged**